### PR TITLE
Fix various typos, try a more accurate ETA calculation, changes to discord status

### PR DESCRIPTION
**Pull request by rekonnected**
1.) Fixed a couple of typos.

2.) Existing ETA calculation tended to be over an hour longer than 2b2t's actual estimated wait time, causing the scheduling function to mis-time joining. Changed the ETA calculation to something more accurate, typically within 10-15 minute delta from actual estimate from server.

3.) Added an option to show which username (account) is currently in the queue, so the user knows which account to connect to the proxy server with, in case they use alts.

4.) Discord bot status wouldn't always update when stopping queue, fixed to make sure it always updates.

You can ignore the commits mentioning changes to .gitignore and docker-compose.yml, they were just changes so that the image would build locally every time I loaded it in WebStorm. I reverted the changes before creating this pull request.

### CODE REVIEW

**Comment by rekonnected** [comment]
Just for context on why I added the username option, I found myself connecting with the wrong account and wanted to check the bot to see which one to connect with. It has its own option in the JSON, enabled by default but can be changed to false as any other setting.

File: main.js [hunk 1]
```diff
@@ -444,14 +450,15 @@ function userInput(cmd, DiscordOrigin, discordMsg) {
 
 function stopMsg(discordOrigin, discordMsg, stoppedThing) {
 	msg(discordOrigin, discordMsg, stoppedThing, stoppedThing + " is **stopped**");
+	dc.user.setActivity(stoppedThing + " is stopped.");
```
Comment (MrGeorgen) [inline main.js:453]: you should use my `activity()` function instead because it checks if the discord bot in enabled

**Review by MrGeorgen — COMMENTED** [review]
I created the ETA calculator originally because the 2b2t ETA was too inaccurate. It worked really well, but was only tested with very long queues from 800 - 1000.

**Comment by themoonisacheese** [comment]
i expect this to crash if the discord bot isn't logged in when it stops queuing. fix this and that's an easy enough merge.

File: main.js (hunk 1 above)
Comment (rekonnected) [reply main.js]: Got it, change made and seems to work right. Thanks!

**Comment by rekonnected** [comment]
Today I tested the system with discordBot: false in config.json, using the direct console commands to connect, and nothing crashed with or without the bot online. If you have the discordBot: true and botToken set and valid, I don't see any reason why the bot wouldn't be connected, so that shouldn't be an issue I believe.

File: main.js (hunk 1 above)
Comment (MrGeorgen) [reply main.js]: When I started working on this project it was the first time I used JS. It nice see you and many others contributing lately

**Review by themoonisacheese — APPROVED** [review]

**Pull request merged**